main.tf

```diff
@@ -50,7 +40,6 @@ resource "azurerm_function_app" "function" {
   https_only                 = true
   client_affinity_enabled    = false
   enable_builtin_logging     = false
-  min_tls_version            = "1.2"
```

Comment:
It seems that `min_tls_version` has been changed to be defined in `site_config`. Can you please move the property definition instead of deleting it?

Reply:
Sure. 1.2 was the default value so i didnt think much of it. Added it again now

Reply:
Thanks! It's perfects!